src/resources/Sources/SourcesSubInterfaces/Information.ts

```diff
@@ -32,12 +32,19 @@ export interface NextOperation {
 
 export interface SourceStatus {
     allowedOperations?: ActivityOperation[];
+    /**
+     * The new type status (includes statuses of transition).
+     */
+    extendedCurrentStatus?: SourceStatusType;
     initialBuild?: boolean;
     numberOfDocuments?: number;
     numberOfProcessedDocuments?: number;
     pausedOnErrorCode?: string;
     refreshType?: OperationType;
     timestamp?: number;
+    /**
+     * The legacy type status (does not include statuses of transition).
+     */
     type?: SourceStatusType;
```

Comment:
is this one getting depreciated?

Reply:
No deprecated, I just wanted to show the difference